src/controller/src/clusters.rs

```diff
@@ -618,6 +619,13 @@ where
                             ));
                         }
 
+                        if let Some(scratch_dir) = &scratch_dir {
+                            args.push(format!(
+                                "--scratch-directory={}",
+                                scratch_dir.to_str().unwrap()
+                            ));
+                        }
+
```

Comment:
I don't think this is correct. We set the `--scratch-directory` in the orchestrator if `disk` is set.